apps/preview/.eslintrc.json

```diff
@@ -0,0 +1,25 @@
+{
```

Comment:
The eslintrc (`.eslintrc.json`) file format was deprecated in favour of flat config files, would require a migration to either one of these:
- eslint.config.js
- eslint.config.mjs
- eslint.config.cjs

https://eslint.org/docs/latest/use/configure/configuration-files